utilities/file_reader.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+
+def read_file(log_file):
+    """
+    Function that accepts log file and opens it.
+    Opens the file in reverse order, iterates through
+    every line to return the logs.
+    """
+    pattern, error_details = re.compile(r'(\d+-\d+-\d+ \d+:\d+:\d+)'), []
```

Comment:
change `pattern` to `timestamp_pattern` and make the variable declarations on different lines.

Reply:
Done.